### PR TITLE
Updated README.md with better vocabulary and removed typos.

### PR DESCRIPTION
**Pull request by LiteBrick204**
## Description
Updated README.md with better vocabulary and removed typos.

## Motivation and Context
Typos were present and needed an amp up

## Types of changes
Typos in the document

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by LiteBrick204** [comment]
Please let me know if this could be better.

File: README.md [hunk 1]
```diff
@@ -46,7 +46,10 @@ UI Refactor(Including Audio Player) -> Add Travis -> Connect a DataBase(FireBase
 * To run the server, use the command `pipenv run python3 server.py`
 
 ### To start the React Frontend
-* Clone the repo using `git clone https://github.com/NJACKWinterOfCode/youtube_cli_flask.git`.
+
+* Open terminal to get started.
+
+* To clone the repo, type `git clone https://github.com/NJACKWinterOfCode/youtube_cli_flask.git` in your terminal.
```
Comment (sansyrox) [inline README.md]: This step is not required now. Can you please delete it?

**Review by sansyrox — COMMENTED** [review]
@LiteBrick204 , all looks good. Please see the inline comments before merge

File: README.md (hunk 1 above)
Comment (LiteBrick204) [reply README.md]: so I should remove the react part? is that what you mean? I do not understand what you're saying. Sorry for the inconvenice.

Comment (sansyrox) [reply README.md]: @LiteBrick204 , just change this url `https://github.com/NJACKWinterOfCode/youtube_cli_flask.git` to the repo url. 😄

**Pull request closed without merging**